pydantic/validators.py

```diff
@@ -161,6 +161,22 @@ def dict_validator(v: Any) -> Dict[Any, Any]:
         return v
 
     with change_exception(errors.DictError, TypeError, ValueError):
+        #
```

Comment:
please remove this. It needs a separate PR and separate description in history, probably with a "breaking change" notice. #558

I guess the test below can go too